README.md

```diff
@@ -6,14 +6,140 @@ A validator for CSV files adhering to the [Data Models](https://github.com/chop-
 
 ## Install
 
-Download the latest binary from the [releases page](https://github.com/chop-dbhi/data-models-validator/releases) for your architecture: Windows, Linux, or OS X.
+Download the latest binary from the [releases page](https://github.com/chop-dbhi/data-models-validator/releases) for your architecture: Windows, Linux, or OS X. The following examples assume the binary has been placed on your `PATH` with the name `data-models-validator`.
 
 ## Usage
 
 Run the following to see the usage.
 
 ```
-data-models-validator -help
+$ data-models-validator -help
+Data Models Validator - 0.1.0-beta.1
```

Comment:
I am typically not a fan of embedding the usage in the README primarily because it will quickly get out of date, especially the version number. I understand that it is for example purposes, but it takes up a lot of space and could later cause confusion. What is correctly being shown is how to get the help/usage using the program.